packages/scaffold/src/partials/w3m-account-tokens-widget/index.ts

```diff
@@ -39,6 +39,10 @@ export class W3mAccountTokensWidget extends LitElement {
   }
 
   // -- Private ------------------------------------------- //
+  private watchConvertValues() {
+    this.watchTokenBalance = setInterval(() => AccountController.fetchTokenBalance(), 1000)
+  }
```

Comment:
won't this spam the api causing a lot of requests to our balance provider? cc @geekbrother

Reply:
Since we have a big TTL for the Zerion cache, we should increase this by at least 5 secs.